packages/tinybird/src/os-client.ts

```diff
@@ -460,6 +460,53 @@ export class OSTinybird {
       }
     };
   }
+
+  getResultForOnDemandCheckHttp() {
+    const parameters = z.object({
+      monitorId: z.number().int(),
+      timestamp: z.number(),
+      url: z.string(),
+    });
+    return async (props: z.infer<typeof parameters>) => {
+      try {
+        const res = await this.tb.buildPipe({
+          pipe: "get_result_for_on_demand_check_http",
+          parameters,
+          data: z.object({
+            latency: z.number().int(), // in ms
+            statusCode: z.number().int().nullable().default(null),
+            monitorId: z.string().default(""),
+            url: z.string().url().optional(),
+            error: z
+              .number()
+              .default(0)
+              .transform((val) => val !== 0),
+            region: z.enum(flyRegions),
+            cronTimestamp: z.number().int().optional(),
```

Comment:
`timestamp`

Reply:
but we are returning the timestamp is the time at with the request was actually triggered 
crontimestamp is the time when we asked for the request to be trigger
IMO we should return the time we asked for the job to run